packages/terra-form-input/src/terra-dev-site/doc/example/InputField.jsx

```diff
@@ -6,10 +6,10 @@ const DefaultInputField = () => (
     inputId="profile-name"
     label="Profile Name"
     help="Note: This can not be changed in the future"
+    type="text"
```

Comment:
This is a non-passive change, methinks. For the existing users who have the current code something on the lines of snippet below, this change breaks the functionality because the default value of `undefined` overrides the existing value that is set. 

```
<InputField
  inputId="numeric-input"
  label="Numeric Value"
  help="Note: This can not be changed in the future"
  inputAttrs={{
    placeholder: 'Enter Digits',
    name: 'numeric',
    type: 'number'
  }}
/>
```